src/sail_json_backend/json.ml

```diff
@@ -428,8 +433,8 @@ let extract_source_code l =
   | None -> "Error - couldn't locate func"
 
 let parse_funcl fcl =
-  match fcl with
-  | FCL_aux (FCL_funcl (Id_aux (Id id, _), Pat_aux (pat, _)), _) -> begin
+    match fcl with
+    | FCL_aux (FCL_funcl (Id_aux (Id id, _), Pat_aux (pat, _)), _) -> begin
```

Comment:
Still a couple of innocuous indentation changes that snuck in. Could you revert the extra spaces added for these two lines?